extensions/EXT_texture_norm16/extension.xml

```diff
@@ -64,8 +70,31 @@ interface EXT_texture_norm16 {
   const GLenum RGB16_SNORM_EXT = 0x8F9A;
   const GLenum RGBA16_SNORM_EXT = 0x8F9B;
 };
+
+enum TextureNorm16SupportedInputTypes {
+  "arraybufferview",
+  "imagebitmap",
+  "imagedata",
+  "htmlimageelement",
+  "htmlcanvaselement",
+  "htmlvideoelement",
+  "offscreencanvas"
+};
   </idl>
 
+  <newfun>
+    <function name="getSupportedInputs" type="sequence&lt;DOMString&gt;">
```

Comment:
Can the return type be sequence&lt;TextureNorm16SupportedInputTypes&gt; ? Could you verify in Chrome that that IDL will parse?

Reply:
Oops. Verified. Also example from WebGLPowerPreference